opcache.php

```diff
@@ -28,9 +28,9 @@
 class OPcache_dashboard {
 	static $instance;
 
-	const PHP_URL = 'http://php.shield-9.org';
+	const PHP_URL = 'http://php.net/manual/en/';
```

Comment:
For other language users, please use 'http://php.net'.

php.net returns same content for these URLs:
* "'http://php.net/XXX" (Language you selected before or English)
* "http://php.net/manual/YY/XXX" (Language in URL string)
* "http://php.net/manual/YY/XXX.php" (Language in URL string)